metaflow/plugins/aws/aws_utils.py

```diff
@@ -88,7 +89,7 @@ def get_docker_registry(image_uri):
     return registry
 
 
-def compute_resource_attributes(decos, compute_deco, resource_defaults):
+def compute_resource_attributes(decos, compute_deco, step_name, resource_defaults):
```

Comment:
I may have to make changes internally to be compatible with this. Not hard though.